### DISCUSSION TITLE
Meetupy kwiecień - czerwiec 2021

### DISCUSSION THREAD

**Discussion by kaluzaaa** [AzureCommunityPL/group#250] [Wydarzenia]
Powolnymi kroki idzie wiosna, przyroda budzi się do życia, odradza zatem czas też odrodzić nasza grupę Azure.
Poszukujemy ochotniczki/ochotników do zorganizowania meetupow online w miesiącach:

- kwiecień
- maj
- czerwiec

Z naszej strony oferujemy:

- platformę do streamu
- wrzucenie napisanych przez Was tekstów na social media na kanały grupy
- platformę do ankiety CFP

Co trzeba zrobić:

- wyznaczyć daty
- znaleźć prelegentów i tematy sesji - są pewne restrykcje co do zakazu promowania firm, płatnych kursów poprzez sesje itp itd, ale o tym można pogadać osobiście
- przeprowadzić CFP
- poprowadzić spotkanie - wprowadzenie, początkowe slajdy i ogólnie bycie frontmanem (lub frontwomanem żeby być diverse)

Osoby zainteresowane zaangażowaniem się w organizacje proszę o potwierdzenie chęci aktywności w wątku, a wtedy zostaną dodane do osobnego kanału dla organizatorów.

CC: @wi5nia

**Comment by PawelHaracz** [comment]
+1

**Comment by miqm** [comment]
✋🏻
